python/mlp/end_effector/limb_rrt_optimized.py

```diff
@@ -27,7 +27,7 @@
 # store the last limbRRT path ID computed
 current_limbRRT_id = None
 
-VERBOSE = False
+VERBOSE = 1 # 0 : disabled, 1: only weight / num try, 2: all path infos, 3: quadprog inputs / outputs
```

Comment:
This could be the good occasion to use https://docs.python.org/3/library/logging.html and `log.warning()` for 1, `log.info()` for 2 & `log.debug()` for 3

Reply:
Thanks for the suggestion, I made this change in all the files. I don't like the fact that some important (but normal for the execution) informations are labelled 'warning'. I will probably add a custom level later.